### PR TITLE
scutil, screencapture, systemsetup, readlink, softwareupdate: add pt_PT translation

### PR DESCRIPTION
**Pull request by rubenandre**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_PT/osx/readlink.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# readlink
+
+> Segue um link simbólico e obtêm a sua informação.
+> Mais informação: <https://www.gnu.org/software/coreutils/readlink>.
```
Comment (navarroaxel) [inline pages.pt_PT/osx/readlink.md:4]:
```suggestion
> Mais informações: <https://www.gnu.org/software/coreutils/readlink>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Sorry for the delay for the review !

File: pages.pt_PT/osx/readlink.md (hunk 1 above)
Comment (rubenandre) [reply pages.pt_PT/osx/readlink.md]: fixed

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @rubenandre

**Pull request merged**